config/config.go

```diff
@@ -1,31 +1,32 @@
-package revel
+package config
 
 import (
 	"errors"
-	"github.com/robfig/config"
 	"path"
 	"strings"
+
+	"github.com/robfig/config"
 )
 
 // This handles the parsing of app.conf
 // It has a "preferred" section that is checked first for option queries.
 // If the preferred section does not have the option, the DEFAULT section is
 // checked fallback.
-type MergedConfig struct {
+type Context struct {
 	config  *config.Config
 	section string // Check this section first, then fall back to DEFAULT
 }
 
-func NewEmptyConfig() *MergedConfig {
-	return &MergedConfig{config.NewDefault(), ""}
+func NewContext() *Context {
+	return &Context{config.NewDefault(), ""}
 }
 
-func LoadConfig(confName string) (*MergedConfig, error) {
+func LoadContext(confName string, confPaths []string) (*Context, error) {
 	var err error
-	for _, confPath := range ConfPaths {
+	for _, confPath := range confPaths {
 		conf, err := config.ReadDefault(path.Join(confPath, confName))
 		if err == nil {
-			return &MergedConfig{conf, ""}, nil
+			return &Context{conf, ""}, nil
```

Comment:
> Since we are modifying LoadConfig should we not pass in the section as well instead of defaulting it to "" ? 

@notzippy Doesn't matter. Lack of section string is [interpreted](https://github.com/robfig/config/blob/master/type.go#L106) as `DEFAULT` anyway.

> Also small peeve of mine is initialization of structures without field names return &Context{conf,""} imho should be return &Context{config:conf}

Ideally we should also `go lint` the code, cover by comments, and add more tests. But to my view of thinking we should keep PRs simple (one change = one PR). Though I'm not always following this rule.